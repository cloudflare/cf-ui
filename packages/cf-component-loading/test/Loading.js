const React = require('react');
const assertEqualJSX = require('assert-equal-jsx');
const Loading = require('../src/Loading');
const Icon = require('cf-component-icon');

describe('Loading', function() {
  it('should render', function() {
    assertEqualJSX(
      <Loading />,
      <Icon type="loading" label="Loading" role="status" />
    );
  });
});
