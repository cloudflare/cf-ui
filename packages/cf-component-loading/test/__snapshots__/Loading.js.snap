// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<i
  className="cf-1isqwou"
>
  <span
    className="cf-1u9fru1"
  >
    Loading
  </span>
</i>
`;
